repos/jsutils/src/node/cmd.js

```diff
@@ -2,13 +2,17 @@
 
 'use strict'
 
-const { logData, setLogs } = require('../log')
-const { promisify, wait } = require('../promise')
+const {
```

Comment:
A number of the methods in this file are replaced by the span-Cmd repo.

Might be a good idea to remove any that are, to try and keep code size down

Reply:
While some should be removed, it's not part of this ticket in any way, so only do it if you think it's worth it.

Reply:
Okay, I removed the `cmd` command since `spawn-cmd` supersedes it. But I didn't see any other functions replaced. I see `getArgs` has an identically named function in `spawn-cmd` but they are actually different.

Reply:
Yah, getArgs was a very very early version of my idea for the args-parse repo.

So it is to getArgs what spawn-Cmd is to Cmd 

So really it could be removed as well

Reply:
Actually, pretty sure this whole fine can be removed. 
All these functions are handled by other repos

Eventually what I would like to do is create a keg-utils repo, that moves a number of items from the the keg-cli as well as a few other place into one repo.

Then a number of the methods there could be moved into here, which would make them re-usable.

But that takes time like anything else. So for now just remove this file. It's not needed

Reply:
sounds good. It's removed